Quick start not working: "globby not found"

I followed the [quickstart](https://gqless.dev/docs/getting-started) in the docs. When trying to run the script to generate the code, I got an error:

```
(node:85825) [MODULE_NOT_FOUND] Error Plugin: @gqless/cli: Cannot find module 'globby'
module: @oclif/config@1.14.0
task: not loading commands, globby not found
plugin: @gqless/cli
```

I was able to work around by running `npm install globby`, but it seems like this should be a dependency of the package itself.

```
"@gqless/cli": "0.0.1-alpha.30"
"gqless": "0.0.1-alpha.28"
```

--- a/.changeset/odd-parents-destroy.md
+++ b/.changeset/odd-parents-destroy.md
@@ -1,0 +1,5 @@
+---
+'@gqless/cli': patch
+---
+
+add globby to dependencies

--- a/packages/cli/package.json
+++ b/packages/cli/package.json
@@ -21,6 +21,7 @@
     "@oclif/config": "^1.14.0",
     "@oclif/plugin-help": "^2.2.3",
     "cosmiconfig": "^6.0.0",
+    "globby": "^11.0.0",
     "got": "^10.6.0",
     "mkdirp": "^1.0.3",
     "prettier": "^1.19.1",

